setup.py

```diff
@@ -72,7 +72,7 @@ def run_tests(self):
         "Programming Language :: Python :: 3.9",
         "Topic :: Software Development :: Libraries :: Python Modules"],
     install_requires=[
-        "oidcmsg==1.4.0",
+        "oidcmsg==1.5.0",
```

Comment:
finally, wow!

Reply:
Latest oidcmsg version is 1.5.1 we should probably point to that.

Reply:
may we have an additional commit in this PR for achieve this?

Reply:
Yes! Will do.